server/helpers/format-user-data.js

```diff
@@ -27,5 +42,11 @@ module.exports = (goalsArr) => {
       }
     });
   });
+  return formattedGoals;
+};
```

Comment:
this is quite confusing, and it took me a while to understand what it was doing. I get it now. 

I think we could probs move the bit where we concatenate the ratings into a separate function. Also, the whole thing could be done with a combination of map / filter, which I think would be easier to understand.

Like:

```js
const matchingRatings = ratings.filter(/* find ratings with correct goal_uid*/);
goals = goals.map(goal => ({ ...goal, ratings: matchingRatings..., });
```
^^ something like that!